pkg/manifest/manifest.go

```diff
@@ -112,7 +112,7 @@ func (man *ManifestInterpreter) CreateResources(conf *contracts.AzionApplication
 			}
 			updated, err := clientOrigin.Update(ctx, conf.Application.ID, OriginKeys[origin.Name], requestUpdate)
 			if err != nil {
-				return err
+				return fmt.Errorf(msg.ErrorUpdateOrigin.Error(), err.Error())
```

Comment:
```suggestion
				return fmt.Errorf("%w: %s", msg.ErrorUpdateOrigin, err.Error())
```

Reply:
here I believe that is a good way to keep the type of error for doing asserts in the future.